blocks/library/image/index.js

```diff
@@ -95,7 +95,7 @@ registerBlock( 'core/image', {
 		return (
 			<figure className="blocks-image">
 				<img src={ url } alt={ alt } />
-				{ caption || !! focus ? (
+				{ caption.length || !! focus ? (
```

Comment:
Yeah the `! caption` check is probably not enough since the caption shape changed.

`! caption || ! caption.length` to match the edit?